### PR TITLE
Added: Bulk edit assign vendor

### LINKED ISSUE #581: Bulk edit change vendor
Currently there is no option to bulk edit change vendor for products. It would be good to add this. 

Would require 

- new bulk action method added to drop down 
- new processing method to process the bulk action

### PR DESCRIPTION
**Pull request by digitalchild**
### All Submissions:

* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This adds a new bulk edit action to assign vendors to multiple products at a time

Closes #581.

### How to test the changes in this Pull Request:

1. Select multiple products in wp-admin 
2. Select edit from bulk actions menu
3. Change the vendor
4. Update 
5. Vendor should be updated on multiple products

### CODE REVIEW

File: classes/admin/class-product-meta.php [hunk 1]
```diff
@@ -297,6 +303,69 @@ public function save_vendor_quick_edit( $product ) {
...
+	public function display_vendor_dd_bulk_edit() {
+
+		global $post;
+		$selected = '';
+
+		$roles     = array( 'vendor', 'administrator' );
+		$user_args = array( 'fields' => array( 'ID', 'display_name' ) );
+
+		$output = "<select style='width:200px;' name='vendor' class='select'>\n";
+		$output .= '<option value="">'. __('— No change —', 'wc-vendors') . '</option>';
```
Comment (dinhtungdu) [inline classes/admin/class-product-meta.php]: missing one space at `'<option value="">'.`

File: classes/admin/class-product-meta.php [hunk 2]
```diff
@@ -279,10 +283,12 @@ public function display_vendor_dd_quick_edit() {
 
 	/*
 	*	Save the vendor on the quick edit screen
+	*
```
Comment (dinhtungdu) [inline classes/admin/class-product-meta.php]: missing param comment

File: classes/admin/class-product-meta.php [hunk 3]
```diff
@@ -297,6 +303,69 @@ public function save_vendor_quick_edit( $product ) {
 		return $product;
 	}
 
+	/*
```
Comment (dinhtungdu) [inline classes/admin/class-product-meta.php]: Comment should be started with `/**`
Leading spaces should be removed.

File: classes/admin/class-product-meta.php [hunk 4]
```diff
@@ -297,6 +303,69 @@ public function save_vendor_quick_edit( $product ) {
...
+	*	Display the vendor drop down on the bulk edit screen
+	*
+	* @since 2.1.14
+	* @version 2.1.14
+	*/
+	public function display_vendor_dd_bulk_edit() {
```
Comment (dinhtungdu) [inline classes/admin/class-product-meta.php]: I must read the whole function and add actions to find out `dd` stands for `dropdown`. IMO this should be explicit.

File: classes/admin/class-product-meta.php (hunk 1 above)
Comment (digitalchild) [reply classes/admin/class-product-meta.php]: Updated

File: classes/admin/class-product-meta.php (hunk 4 above)
Comment (digitalchild) [reply classes/admin/class-product-meta.php]: Updated.

File: classes/admin/class-product-meta.php (hunk 3 above)
Comment (digitalchild) [reply classes/admin/class-product-meta.php]: Updated

**Pull request merged**